src/lib/userStorage/useFeedReady.js

```diff
@@ -0,0 +1,25 @@
+import { useCallback } from 'react'
+import usePropsRefs from '../hooks/usePropsRefs'
+import { useUserStorage } from '../wallet/GoodWalletProvider'
+
+export const onFeedReady = async userStorage => {
+  const { feedStorage, initializedRegistered } = userStorage
+
+  if (!initializedRegistered) {
+    await feedStorage.ready
+  }
+}
+
+const useFeedReady = () => {
+  const userStorage = useUserStorage()
+  const { initializedRegistered } = userStorage
+  const [getStorage] = usePropsRefs([userStorage])
+
+  const onReady = useCallback(async () => {
+    await onFeedReady(getStorage())
+  }, [getStorage])
+
+  return [initializedRegistered, onReady]
+}
```

Comment:
the userstorage item doesnt refresh when initializeregistered is set.
i dont think this hook will work correctly if it loads before initializeregistered is set
@serdiukov-o-nordwhale

Reply:
please keep things simple! there's no need to hookify everything

Reply:
fixed: https://github.com/GoodDollar/GoodDAPP/commit/4bcc2347965b07d8774b7343d9bdbcc42e53fa92